src/validation.cpp

```diff
@@ -1851,6 +1851,18 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    // SEGWIT2X signalling.
+    if ( VersionBitsState(pindex->pprev, chainparams.GetConsensus(), Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE &&
```

Comment:
@tomasvdw My point is that locked-in period makes perfect sense when "activation" means a feature (like segwit).  It makes less sense if BIP91 "activation" just means starting signaling for BIP141 activation.  This is why I was advocating separating the start-signaling date from the start-orphaning date.